sql/ddl_api.sql

```diff
@@ -194,6 +194,12 @@ CREATE OR REPLACE PROCEDURE distributed_exec(
        transactional BOOLEAN = TRUE)
 AS '@MODULE_PATHNAME@', 'ts_distributed_exec' LANGUAGE C;
 
+-- Execute create_restore_point() on each data node
+CREATE OR REPLACE FUNCTION create_distributed_restore_point(
+    name                     NAME
```

Comment:
Yes, but restore point name has also limit by name size (which is 64 chars max). I can switch to TEXT and add additional check for length in such case

Reply:
I think we should use same type as `pg_create_restore_point`, which uses `TEXT`. As put in the PostgreSQL docs: `The name type exists only for the storage of identifiers in the internal system catalogs and is not intended for use by the general user.`. Since the restore point is not a PG identifier I think `TEXT` is appropriate here.